### ISSUE TITLE
inline refactor bug in jsx

### ISSUE THREAD

**Issue by skt-t1-byungi** [p42ai/refactor-vscode#15]
```jsx
export function App() {
  const handleClick = () => console.log(test);
  return (
    <>
      <div onClick={handleClick}>test</div>
    </>
  );
}
```

If the `handleClick` function is refactored inline, the output is as follows. (`<`,`t` is repeated twice )

```jsx
export function App() {
  return (
    <>
      <<div onClick={() => console.log(test)}>ttest</div>
    </>
  );
}
```

**Comment by lgrammel** [comment]
Thanks for the bug report!

I've tried the example that you have provided and inlining `handleClick` works for me:

https://user-images.githubusercontent.com/205036/156144750-bcd79465-0bb1-4dfa-ab82-bce60e9aca23.mov

Which P42 version are you using? Were there any syntax errors in the file that you were refactoring?

**Comment by skt-t1-byungi** [comment]
![d2](https://user-images.githubusercontent.com/16894698/156148198-08a1ef15-2dc1-4527-8eca-b95ae4335e7a.gif)
The version is 1.86.1.
I tested with the same code.
I am windows os and I use many other extensions as well.
(I'm busy, I'll test later by deleting other extensions. 😅)

**Comment by lgrammel** [comment]
Thanks, I was able to reproduce it. It is caused by the `\r\n` end of line sequence under windows. I will prepare a fix.

**Comment by lgrammel** [comment]
@skt-t1-byungi  I have just released version `1.86.2`, which fixes the bug. Please re-open this issue if you still encounter the bug in `1.86.2` or higher. Thanks again for the bug report!
